### DISCUSSION TITLE
Adding a starred repository to a list refreshes the view before extra

### DISCUSSION THREAD

**Discussion by pcider** [community/community#133029] [Mobile]
1. Goto your starred Repositories.
2. Scroll all the way down until you see a loading throbber. Take note of the last repository before needing to fetch/load the next "page" of starred repositories. Lets call it Repo X.
3. Add any repository after Repo X to any list.
4. After tapping out of the list selection prompt, the view goes back to Repo X.

**Comment by pineda0021** [comment]
1. Go to your starred Repositories.
2. Scroll down until you see a loading throbber.
This indicates that more repositories are being loaded.
3. Take note of the last repository before needing to fetch/load the next "page" of starred repositories.
Let's call this repository Repo X.
4. Add any repository after Repo X to any list.
This involves selecting a repository and adding it to a specific list (e.g., a custom collection or category).
5. After tapping out of the list selection prompt, the view goes back to Repo X.
This is the unexpected behavior. Instead of maintaining your current position, the interface refreshes and returns to the position of Repo X.
This behavior can be frustrating, especially if you have a long list of starred repositories and are trying to organize them efficiently. If you haven't already, you should report this issue to the support team of the repository hosting service you are using (e.g., GitHub). They can provide a fix or workaround for this UI issue.
